extensions/sql-migration/src/models/loginMigrationModel.ts

```diff
@@ -179,7 +180,25 @@ export class LoginMigrationModel {
 			}
 		}
 
-		this.errorCountMap.set(LoginMigrationStep[step], errorBuckets);
+		// Making a string of the map elements of errorBuckets
+		var errorBucketsString = JSON.stringify(
```

Comment:
I recommend using const (preferred)  or let over var in typescript

Reply:
changed to const in latest commit.